src/subscription/subscribe.js

```diff
@@ -96,6 +107,16 @@ export function subscribe(
     );
 }
 
+// This function checks if the error is a GraphQLError. If it is, convert it to
```

Comment:
Nits:

Use /* for multi-line doc blocks.

Also, it would be nice to move this fn lower in the file - keeping `subscribe()` and `subscribeImpl()` directly next to each other in the file.